### PR TITLE
Rewrite layout grids to avoid header full-width  breakout.

### PR DESCRIPTION
**Pull request by notlee**
- Fixes a layout issue in Chrome 80
- Prevents horizontal scroll when scrollbars are enabled

**Prevents horizontal scroll when scrollbars are enabled. @chee pointed this issue out a while ago.**
Before/After
Screenshot 2020-02-11 at 16 52 21
Screenshot 2020-02-11 at 16 52 17

**Fixes a layout issue in Chrome 80**
Screenshot 2020-02-11 at 16 52 56
Screenshot 2020-02-11 at 16 52 56 1
Screenshot 2020-02-11 at 16 53 05
Screenshot 2020-02-11 at 16 53 10
Screenshot 2020-02-11 at 16 53 20
Screenshot 2020-02-11 at 16 53 23
Screenshot 2020-02-11 at 16 53 27
Screenshot 2020-02-11 at 17 02 39
Screenshot 2020-02-11 at 17 02 53
Screenshot 2020-02-11 at 17 03 12

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-layout bundle size difference from 4.0.9 to 5aaca48377b3ddc80933448f88d19317c10cf99e
css, internal: 0.68kb increase (0.06kb/gzip)
An insignificant difference was also found for: js

**Comment by notlee** [comment]
Regression: there's an extra grid-gap on the query layout if the sidebar is not used

File: src/scss/_grid.scss [hunk 1]
```diff
@@ -60,12 +48,15 @@
...
-			"header"
-			"hero"
-			"main"
-			"footer";
+			"header header header"
+			". hero ."
+			". main ."
+			"footer footer footer";
 		grid-template-rows: auto auto 1fr auto;
+		// 0 column width ro create a grid gap the header and footer can span
```
Comment (notlee) [inline src/scss/_grid.scss]:
```suggestion
```

**Comment by origamiserviceuser** [comment]
o-layout bundle size difference from 4.0.9 to a114f4048bd0fe3d79f74eb18ebe634b76cbd411
css, internal: 0.68kb increase (0.06kb/gzip)
An insignificant difference was also found for: js

**Comment by notlee** [comment]
@rowanmanning it was never the cause but removing our redundant `minmax` functions in the dev console appeared to render correctly in Chrome; I'm not sure what about the last PR led to the horizontal scrolling

**Comment by origamiserviceuser** [comment]
o-layout bundle size difference from 4.0.9 to bbe2913d6bad3dc3c536a941af4dd3fd0577a54b
css, internal: 0.68kb increase (0.06kb/gzip)
An insignificant difference was also found for: js

**Pull request merged**